multimedia/tvheadend/Makefile

```diff
@@ -28,110 +24,205 @@ include $(INCLUDE_DIR)/nls.mk
 define Package/tvheadend
   SECTION:=multimedia
   CATEGORY:=Multimedia
-  TITLE:=Tvheadend is a TV streaming server for Linux
-  DEPENDS:=+libopenssl +librt +zlib +libffi +TVHEADEND_AVAHI_SUPPORT:libavahi-client $(ICONV_DEPENDS)
+  TITLE:=tvheadend
```

Comment:
Previous title was OK. No need to change it. This title is shown e.g. in `make menuconfig`

Reply:
Misleading. It's not just for Linux.
How about "TV streaming server"?